2-copy-of-code/lesson-13/scripts/amazon.js

```diff
@@ -72,14 +72,14 @@ document.querySelectorAll('.js-add-to-cart')
       const quantitySelector = document.querySelector(
         `.js-quantity-selector-${productId}`
       );
-      const quantity = quantitySelector.value;
+      const quantity = Number(quantitySelector.value);
```

Comment:
Hello, I encounter an issue while test the addToCart function. Jasmine showed an error "Can not read property null (reading 'value')"   which I  traced back to this line in my code. What could have caused it?